playlistmanager.lua

```diff
@@ -389,7 +417,7 @@ function get_name_from_index(i, notitle)
   local title = mp.get_property('playlist/'..i..'/title')
   local name = mp.get_property('playlist/'..i..'/filename')
 
-  local should_use_title = settings.prefer_titles == 'all' or name:match('^https?://') and settings.prefer_titles == 'url'
+  local should_use_title = settings.prefer_titles == 'all' or is_protocol(name) and settings.prefer_titles == 'url'
```

Comment:
I think this one should apply only to http and https protocols. I don't think youtube-dl will work with anything else.

Reply:
Oh actually nevermind, it's about using the title instead of fetching it. This should be fine to for any protocol.